iOSBuildTests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr --output utr
+chmod +x utr
+EDITOR_LOCATION=
```

Comment:
Remove this entirely. It will expand to empty string anyway and will open the opportunity to set via env variable.